core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DriverComponent.scala

```diff
@@ -40,23 +41,6 @@ trait DriverComponent {
     }
   }
 
-  //in general, we only support alphanumeric, spaces, _, and - for user-input
```

Comment:
Moved to `StringUtils` because I want to use `validateUserDefinedString` outside of a DB context